roles/swift-common/defaults/main.yml

```diff
@@ -123,6 +123,8 @@ swift:
         criticality: 'critical'
       fstab_mounts:
         criticality: 'critical'
+      check_swift_dispersion:
+        criticality: 'critical'
```

Comment:
.sh will not be used anymore. only has 1 check for swift-dispersion, and related json file's content is changed (command is changed to .py ).
For the interval, let me change it to 2minutes.